src/styles/globals.css

```diff
@@ -6,4 +6,15 @@ body,
 #__next,
 html {
   height: 100%;
+  font-family: "inter", sans-serif;
+  font-size: 62.5%;
```

Comment:
https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/#the-sixtwofive-trick-13

Might not be that good idea